### ISSUE TITLE
Pre-trained model about issue #22

### ISSUE THREAD

**Issue by yitang-hnu** [hasanirtiza/Pedestron#28]
@hasanirtiza. Thanks for your detailed explanation for pre-training and fin-tuning! It really works, and I got the result of 8.53 MR^-2 in CityPerson, but it is still 1 MR-^2 point behind your best result. I have tried a series of learning rate(from 0.0025 to 0.0005 for 1 img and 1 GPU) for fine-tuning, and only got the result of 8.53 MR^-2 in CityPerson. So, I think it is not very well of my pre-trained model on Wilder Pedestrian and ECP before fine-tune. Could you provide the model that you pre-trained on Wilder Pedestrian and ECP before fine-tuning step in your paper Sec 6.2?

BTW, from my experiments on CityPerson datasets, the model pre-trained on Wilder Pedestrian only is much better than the model pre-trained on WIDER PERSON and WIDER Pedestrian that you provided on CityPerson datasets. If it is convenient, could you provide the pre-trained model only on WIDER Pedestrian? I think it is also much helpful for generic human detection.

**Comment by hasanirtiza** [comment]
Yes, I think we can provide the pre-trained models, it will take some time. As far as your 8.5 vs _ours_ 7.5 MR^-2 is concerned, I mean its in the same ballpark as reported in the paper. I would not too much worry about < =~1% of MR-^2 points. In anyways, I think we will try to provide the pre-trained models.

Secondly, can you share with us the converted annotations that you use for training Pedestron for Wider Person dataset ? Perhaps upload a google link or you can directly email at ihasan085@gmail.com

**Comment by yitang-hnu** [comment]
Thank you and looking forward to your pre-trained model! 
Maybe my explanation misunderstood you. I mean, the model I pre-trained on wider pedestrian by myself can get a little better result(about 0.3 MR^-2) than the model you provided in the readme.md(the part Benchmark→Pre-Trained models→ Cascade Mask R-CNN→6. WIDER PERSON and WIDER Pedestrian). I guess it is the model pre-trained on WIDER PERSON and WIDER Pedestrian from the subtitle.
Additionally, I am also interested in WIDER PERSON, and if I get the the converted annotations, I will sent it to you by email.

**Comment by hasanirtiza** [comment]
I see, let me clarify the model in the README.md is only trained on WiderPedestrian. I think the description is misleading over there.

**Comment by yitang-hnu** [comment]
Thanks, I get it. I am looking forward your pre-trained model. Good luck with your study!
